textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexMatcherTestHelper.java

```diff
@@ -26,7 +26,7 @@
 public class RegexMatcherTestHelper {
 
     RegexMatcher regexMatcher;
-    IDataWriter dataWriter;
+    DataWriter dataWriter;
```

Comment:
@qinhuah Since @zuozhiw changed `IDataWriter` to `DataWriter`, please make the same change to other places so that they are consistent.